process/suspension.html

```diff
@@ -48,7 +48,7 @@ <h4>Nearby</h4>
     <div id="Content">
 
 <p>The W3C process document empowers the Director to <a href="https://www.w3.org/Consortium/Process/#ParticipationCriteria">suspend or remove participants from groups</a>. This document provides details about implementation and delegation.</p>
-
+<p>The  <a href="/Consortium/cepc/">W3C Code of Ethics and Professional Conduct</a> takes priority over any inconsistencies in this document.</p>
```

Comment:
```suggestion
<p>The  <a href="/Consortium/cepc/">W3C Code of Ethics and Professional Conduct</a> takes priority in case of any inconsistencies with this document.</p>
```